### PR TITLE
Added link to a sign-up form.

### PR DESCRIPTION
**Pull request by dsumin**

### CODE REVIEW

File: public/js/app/templates/sessionNewTemplate.handlebars [hunk 1]
```diff
@@ -28,6 +28,9 @@
       {{#if errors}}
         {{#link-to "forgot-password"}}Forgot your password?{{/link-to}}
       {{/if}}
+
+      <p>New to freefeed? <a href="https://docs.google.com/forms/d/1P4cyYUZlxABbWvZrM5jHRcMUFQFpc8qfbZWjGulobsA/viewform?c=0&w=1
```
Comment (clbn) [inline public/js/app/templates/sessionNewTemplate.handlebars:32]: freefeed -> FreeFeed

File: public/js/app/templates/sessionNewTemplate.handlebars [hunk 2]
```diff
@@ -28,6 +28,9 @@
...
+" target="_blank">Create an account »</a></p>
```
Comment (clbn) [inline public/js/app/templates/sessionNewTemplate.handlebars:33]: I'd prefer not to use "»" as an arrow. At least we are not doing it anywhere else on the site.

Comment (dsumin) [inline public/js/app/templates/sessionNewTemplate.handlebars:33]: > I'd prefer not to use "»" as an arrow. At least we are not doing it anywhere else on the site.

That was used by the original pepyatka. I don't see any issues with that.

File: public/js/app/templates/sessionNewTemplate.handlebars (hunk 1 above)
Comment (dsumin) [inline public/js/app/templates/sessionNewTemplate.handlebars:32]: Page header uses "freefeed". I would hate to see "freefeed" and "FreeFeed" on the same page.

**Comment by clbn** [comment]
OK, one is "design" question, another is "branding" or sorf of. Since I'm not feeling, well, "authorized" to give :-1: on both of these matters, I give up.

**Pull request merged**